pkgs/development/libraries/libproxy/default.nix

```diff
@@ -28,6 +28,10 @@ stdenv.mkDerivation rec {
     hash = "sha256-pqj1LwRdOK2CUu3hYIsogQIXxWzShDuKEbDTbtWkgnQ=";
   };
 
+  patches = [
+    ./fix-build-on-darwin.patch
```

Comment:
Would be nice to include more detailed description of the build failure in the commit message so that in the future, we can check if the patch is still relevant.

Reply:
I think I'll send the patch upstream and create a new PR for libproxy in nixpkgs later today, since Midori is crashing on macOS...

Reply:
Moved to #185853.